recipes/cookbook_uploader.rb

```diff
@@ -64,7 +72,7 @@
 end
 
 # This is necessary in order to create Jenkins jobs with security enabled.
-node.run_state[:jenkins_private_key] = secrets['jenkins_private_key']
+node.run_state['jenkins_private_key'] = secrets['jenkins_private_key']
```

Comment:
This will cause jenkins permissions to fail. It needs to be a symlink. I had to debug this before.